proto/src/lib.rs

```diff
@@ -1,6 +1,6 @@
 //! tendermint-proto library gives the developer access to the Tendermint proto-defined structs.
 
-#![no_std]
+#![cfg_attr(not(feature = "grpc"), no_std)]
```

Comment:
This effect could be made explicitly seen also in `Cargo.toml` by making this conditional on the "std" feature, which "grpc" would depend on. In practice though, "grpc" is the only feature that requires std for now.